backend/battles/views.py

```diff
@@ -73,3 +73,16 @@ def get_context_data(self, **kwargs):  # noqa
             Q(user_creator=user) | Q(user_opponent=user), settled=True
         )
         return context
+
+
+class ListActiveBattlesView(LoginRequiredMixin, ListView):
```

Comment:
Are the tests for this view coming in another PR? :eyes:

Reply:
Will be!!!